verilog/analysis/verilog_linter.cc

```diff
@@ -383,22 +383,48 @@ void GetLintRuleDescriptionsHelpFlag(std::ostream* os,
   }
 }
 
-void GetLintRuleDescriptionsMarkdown(std::ostream* os) {
+void GetLintRuleFile(std::ostream *os) {
+  // Set up the map.
+  auto rule_map = analysis::GetAllRuleDescriptions();
+  for (const auto &rule_id : analysis::kDefaultRuleSet) {
+    rule_map[rule_id].default_enabled = true;
+  }
+
+  // Print all the rules
+  for (const auto &rule : rule_map) {
+    *os << (rule.second.default_enabled ? "+" : "-");
+    *os << rule.second.descriptor.name;
+
+    bool first = true;
+    for (auto it = rule.second.descriptor.param.begin();
```

Comment:
Can you make this a range-based loop ? `for (const auto &rule_param : rule.second.descriptor.param) {...}`

(this is what the clang-tidy CI check is about but for some reason the github UI is brokken, so you can't see the log output in the CI run, but by looking at the [raw log](https://pipelinesghubeus23.actions.githubusercontent.com/Vjp6wX1uV2r1O1Bj6h2lKqrBcOftZCnAHGZLuirThVBBZtBnxy/_apis/pipelines/1/runs/5563/signedlogcontent/6?urlExpires=2024-03-23T16%3A18%3A47.0220215Z&urlSigningMethod=HMACV1&urlSignature=rVLdOFSuGdtk6YYaxPGUkg2JjYTLTLUvmal%2B4cOCRVE%3D)

Reply:
No problems. Done